experiments/semisynthetic_simulation/docs/multiple_runs.md

```
@@ -0,0 +1,86 @@
+```bash
+python experiments/semisynthetic_simulation/generate_configs.py \
+  my_scenario \
+  --n_runs 50 \
+  --base_seed 42 \
+  --experiments_dir ./outputs/causal/semisynthetic_study/runs \
+  --meds ./path/to/meds/data
+```
+
+This generates one config per run under `generated_configs/`:
+
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)